app/helpers/application_helper.rb

```diff
@@ -159,6 +159,10 @@ def logo_svg
     end
   end
 
+  def show_logo(logo)
```

Comment:
I was wondering if we want to rename this to `logo_url`, since that's what the params represent and what this method returns. Maybe we could even make it a bit more explicit that there's a sensible default, i.e. `safe_logo_url` or something like that.

Reply:
I like this suggestion @citizen428. I've updated 👌🏽